src/v2/options.ts

```diff
@@ -200,10 +200,27 @@ export interface GlobalOptions {
    * @remarks
    * When true, requests with invalid tokens autorespond with a 401
    * (Unauthorized) error.
-   * When false, requests with invalid tokens set event.app to undefiend.
+   * When false, requests with invalid tokens set event.app to undefined.
    */
   enforceAppCheck?: boolean;
 
+  /**
+   * Determines whether Firebase App Check token is consumed on request. Defaults to false.
+   *
+   * @remarks
+   * Set this to true to enable the App Check replay protection feature by consuming App Check token on callable request.
+   * Tokens that are found to be already consumed will return app data as alreadyConsumed.
```

Comment:
We might also want to mention that setting this field to `true` does not affect the behaviour of `enforceAppCheck`.

> This option does not affect the `enforceAppCheck` option. Setting the latter to `true` will cause your Callable Function to auto-respond with a 401 error on invalid tokens. Tokens that are found to be already consumed (but are otherwise valid) will not trigger such an auto-response; instead, in the `event.app` context, the `alreadyConsumed` field will be set to `true`, which your business logic can inspect and react to.